README.md

```diff
@@ -366,13 +366,32 @@ My name is Oleksii. I am 28. The course is very informative, I knew most of it,
 I share you my repo [here](https://github.com/OleksiiPry/kottans-frontend).
 
 # Hey there!
+<<<<<<< HEAD
+﻿﻿### hello!
+=======
 
 ### hello!
 
+>>>>>>> master
 
 **I'm Natalie**
 
 Happy to join this very energetic and inspiring community! Wish all of us to become front-end experts! :art:
+<<<<<<< HEAD
```

Comment:
Looks like you tried to solve the conflicts but the lines with <<<<<< & >>>>>> should be deleted when you are fixing conflicts